### DISCUSSION TITLE
I've downloaded in both Chrome and Edge and it doesn't work.

### DISCUSSION THREAD

**Discussion by feistmann** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#34] [General]
What next?

**Comment by HerMajestyDrMona** [comment]
What do you mean by "doesn't work"? It's hard to help when you can not define what exactly is happening.

Do you see the Tray Icon after you start the program?
https://user-images.githubusercontent.com/24290461/140068775-473a45f7-1f6a-4b10-8664-e5f8d49440ab.png

**Reply by wumpscutde** [reply]
Значок в трее не отображается. Пробовал запуск 2.1.0.0 от имени админа. Никакого результата. Такое впечатление, что вообще не запускается. Антивирусов не установлено, только стандартный защитник.  Пробовал разные версии. винда 11 21H2

**Reply by wumpscutde** [reply]
Значок в трее появился. Но по прежнему запрет на перетаскивание(

**Reply by HerMajestyDrMona** [reply]
How about you translate your messages so everyone who reads it won't need to?

Please try the custom configuration option:
`DetectKnownPixelColorsToPreventAccidentalEvents=0`

Maybe you're using some custom theme.

**Comment by feistmann** [comment]
Hi,

Sorry.  I just bought a new PC with Win 11.  I have quite a number of Excel 
files on my desktop which I also had on the taskbar of my previous Win 8.1 
PC.  I’ve downloaded your program, and when I try to drag and drop one of 
the files on to the taskbar it doesn’t work, and shows the circle with a 
diagonal line through it.

I do see the Tray Icon.

Thanks.

**Reply by HerMajestyDrMona** [reply]
Sorry, I somehow missed your messages :( Mail has never arrived.

Did you manage to solve this issue? Is the LogoWin + T hotkey working for you when you press it manually?

**Comment by HerMajestyDrMona** [comment]
Ah, I know what you mean.

You wanted to pin files to the taskbar, but this feature is not available in this program.

You can read about it here:
https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#limitations

Basically it's not even possible to pin files (except some programs) to the taskbar in Windows 11, using any other method. So I could not add this feature.

This program restores the drag and drop behaviour that only restores the minimized windows when dragging files to their icons.
